tests/test_qnode.py

```diff
@@ -109,6 +110,36 @@ def operable_mock_device_2_wires():
         yield Device(wires=2)
 
 
+class TestQNodeConstruct:
```

Comment:
Is this class name accurate?

Reply:
Perhaps `TestQNodeBestMethod` is more appropriate.

Reply:
Yep that sounds better. Ties into the _best_method, uh, method, as well.